packages/editor/test/Sx.tsx

```diff
@@ -44,243 +49,203 @@ describe('Sx.Typography', () => {
   })
 
   test('edits sx.fontSize', async () => {
     const onChange = jest.fn()
     const tree = render(
-      <Sx.Typography
-        theme={theme}
-        value={style}
-        onChange={onChange}
-      />
+      <Sx.Typography theme={theme} value={style} onChange={onChange} />
     )
     const input = await waitForElement(() => tree.findByLabelText('Font Size'))
     fireEvent.change(input, {
       target: {
         value: '2',
-      }
+      },
     })
     expect(onChange).toHaveBeenCalledWith({ fontSize: 2 })
   })
 
   test('edits sx.fontWeight', async () => {
     const onChange = jest.fn()
     const tree = render(
-      <Sx.Typography
-        theme={theme}
-        value={style}
-        onChange={onChange}
-      />
+      <Sx.Typography theme={theme} value={style} onChange={onChange} />
+    )
+    const input = await waitForElement(() =>
+      tree.findByLabelText('Font Weight')
     )
-    const input = await waitForElement(() => tree.findByLabelText('Font Weight'))
     fireEvent.change(input, {
       target: {
         value: '500',
-      }
+      },
     })
     expect(onChange).toHaveBeenCalledWith({ fontWeight: '500' })
   })
 
   test('edits sx.lineHeight', async () => {
     const onChange = jest.fn()
     const tree = render(
-      <Sx.Typography
-        theme={theme}
-        value={style}
-        onChange={onChange}
-      />
+      <Sx.Typography theme={theme} value={style} onChange={onChange} />
+    )
+    const input = await waitForElement(() =>
+      tree.findByLabelText('Line Height')
     )
-    const input = await waitForElement(() => tree.findByLabelText('Line Height'))
     fireEvent.change(input, {
       target: {
         value: '1.625',
-      }
+      },
     })
     expect(onChange).toHaveBeenCalledWith({ lineHeight: '1.625' })
   })
 
   test('renders without props', async () => {
-    const tree = render(
-      <Sx.Typography />
+    const tree = render(<Sx.Typography onChange={noop} />)
+    const input = await waitForElement(() =>
+      tree.findByLabelText('Line Height')
     )
-    const input = await waitForElement(() => tree.findByLabelText('Line Height'))
     expect(input).toBeTruthy()
   })
 
   test('adds tag name to input name', async () => {
-    const tree = render(
-      <Sx.Typography
-        tag='h1'
-      />
-    )
-    const input = await waitForElement(() => tree.findByLabelText('Font Family'))
+    const tree = render(<Sx.Typography tag="h1" onChange={noop} />)
+    const input = (await waitForElement(() =>
+      tree.findByLabelText('Font Family')
+    )) as HTMLInputElement
     expect(input.name).toBe('styles.h1.fontFamily')
   })
 })
 
 describe('Sx.Margin', () => {
   test('edits sx.ml', async () => {
     const onChange = jest.fn()
-    const tree = render(
-      <Sx.Margin
-        theme={theme}
-        value={style}
-        onChange={onChange}
-      />
+    const tree = render(<Sx.Margin value={style} onChange={onChange} />)
```

Comment:
I had to remove the `theme` prop because the `Sx.Margin` component doesn't accept that in its prop type.